specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizer.json

```diff
@@ -0,0 +1,637 @@
+        "clusters": {
+          "description": "Object mapping ClusterIds to Key lists.",
+          "type": "object",
+          "additionalProperties": {
+            "uniqueItems": false,
+            "type": "array",
+            "items": { "type": "string" }
+          }
+        }
+      }
+    },
+    "ModelResult": {
+      "description": "Result of a model status query operation.",
+      "type": "object",
+      "properties": {
+        "modelId": {
+          "description": "Get or set model identifier.",
+          "type": "string",
+          "format": "uuid",
+          "x-nullable": false
+        },
+        "status": {
+          "description": "Get or set the status of model.",
+          "enum": [ "created", "ready", "invalid" ],
+          "type": "string"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "description": "Get or set the created date time of the model.",
+          "type": "string"
+        },
+        "lastUpdatedDateTime": {
+          "format": "date-time",
+          "description": "Get or set the model last updated datetime.",
+          "type": "string"
+        }
+      }
+    },
+    "ModelsResult": {
+      "description": "Result of query operation to fetch multiple models.",
+      "type": "object",
+      "properties": {
+        "models": {
+          "description": "Collection of models.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ModelResult"
+          }
+        }
+      }
+    },
+    "FormDocumentReport": {
+      "type": "object",
+      "properties": {
+        "documentName": {
+          "description": "Reference to the data that the report is for.",
+          "type": "string"
+        },
+        "pages": {
+          "format": "int32",
+          "description": "Total number of pages trained on.",
+          "type": "integer"
+        },
+        "errors": {
+          "description": "List of errors per page.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "status": {
+          "description": "Status of the training operation.",
+          "enum": [
+            "success",
+            "partialSuccess",
+            "failure"
+          ],
+          "type": "string"
+        }
+      }
+    },
+    "FormOperationError": {
+      "description": "Error reported during an operation.",
+      "type": "object",
+      "properties": {
+        "errorMessage": {
+          "description": "Message reported during the train operation.",
+          "type": "string"
+        }
+      }
+    },
+    "ErrorResponse": {
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorInformation"
+        }
+      }
+    },
+    "ErrorInformation": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string"
```

Comment:
What is the set of code values?

Reply:
20xx-2050 - Client Errors
2500 - internal error
30xx-3500 - backend error